Source/BugsnagCrashSentry.m

```diff
@@ -44,7 +46,7 @@ - (void)install:(BugsnagConfiguration *)config
     
     bsg_kscrash_setHandlingCrashTypes(crashTypes);
     
-    if (![[BSG_KSCrash sharedInstance] install])
+    if (![ksCrash install])
         bsg_log_err(@"Failed to install crash handler. No exceptions will be reported!");
```

Comment:
nit: braces round conditional.  Also, while this is being modified, would a rename to e.g. `tryInstall` read better?  I'm slightly swithering on that.

Reply:
Added braces - I'll keep the name as `install` for now as it reduces the size of the changeset, and it's something we can alter later if needed.